bucket/byenow.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.2",
+    "description": "Utility for folder removal",
```

Comment:
```suggestion
    "description": "A command-line utility for faster folder removal.",
```

Reply:
@h404bi Dot is not needed. It is not sentence. `command line` is obsolete as it is in Main bucket